### DISCUSSION TITLE
UE 5.4.4 Packaging error

### DISCUSSION THREAD

**Discussion by Ineed1name** [pafuhana1213/KawaiiPhysics#142] [Q&A]
Hi there!
I'm trying to package my UE5.4 project and getting following errors when trying to do so:

![Capture](https://github.com/user-attachments/assets/1d938ec9-b4a3-4064-9f4f-cd7becf5bd4f)

LogClass: Error: DoubleProperty FAnimNode_KawaiiPhysics::LastEvaluatedTime is not initialized properly even though its struct probably has a custom default constructor. Module:KawaiiPhysics File:Public/AnimNode_KawaiiPhysics.h
LogClass: Error: BoolProperty FKawaiiPhysics_ExternalForce::bIsOneShot is not initialized properly. Module:KawaiiPhysics File:Public/KawaiiPhysicsExternalForce.h
LogClass: Error: ObjectProperty FKawaiiPhysics_ExternalForce_Wind::World is not initialized properly even though its struct probably has a custom default constructor. Module:KawaiiPhysics File:Public/KawaiiPhysicsExternalForce.h
LogClass: Error: StructProperty FCollisionLimitDataBase::Guid is not initialized properly even though its struct probably has a custom default constructor. Module:KawaiiPhysics File:Public/KawaiiPhysicsLimitsDataAsset.h
LogAutomationTest: Error: LogClass: DoubleProperty FAnimNode_KawaiiPhysics::LastEvaluatedTime is not initialized properly even though its struct probably has a custom default constructor. Module:KawaiiPhysics File:Public/AnimNode_KawaiiPhysics.h [log]
LogAutomationTest: Error: LogClass: BoolProperty FKawaiiPhysics_ExternalForce::bIsOneShot is not initialized properly. Module:KawaiiPhysics File:Public/KawaiiPhysicsExternalForce.h [log]
LogAutomationTest: Error: LogClass: ObjectProperty FKawaiiPhysics_ExternalForce_Wind::World is not initialized properly even though its struct probably has a custom default constructor. Module:KawaiiPhysics File:Public/KawaiiPhysicsExternalForce.h [log]
LogAutomationTest: Error: LogClass: StructProperty FCollisionLimitDataBase::Guid is not initialized properly even though its struct probably has a custom default constructor. Module:KawaiiPhysics File:Public/KawaiiPhysicsLimitsDataAsset.h [log]

Had it working with 5.3 without any issues. Any ideas on how to fix it?
Not the best with c++ so can't really figure what's going on.
Appreciate your help!

**Comment by pafuhana1213** [comment]
Hi @Ineed1name,
Would you please share your enviroment with me ? For example... kawaii physics version, BP project or C++ project, etc. If you can give me all the relevant information I might be able to help you.

thanks!

**Reply by Ineed1name** [reply]
Thanks for your quick reply!
I'm using the latest release of kawaii physics 1.17.1 with a c++ project. When building the project in Visual Studio I don't get any errors.

**Reply by Ineed1name** [reply]
oh and I have the plugin inside of the project folder, if that is of any help

**Reply by th020113** [reply]
I also encountered the same error on version 1.17.1. The error doesn't occur if I downgrade to 1.16.0. I'm using UE 5.4.4 too.

**Comment by pafuhana1213** [comment]
Hi guys, sorry for my very very late reply.

I've committed https://github.com/pafuhana1213/KawaiiPhysics/commit/558179031618a247debc9b597e06539381d93eae to fix it now.
And, it will be included to next release is coming soon.

thanks!
